src/dvclive/lightning.py

```diff
@@ -50,35 +50,19 @@ def _should_call_next_step():
 
 
 class DVCLiveLogger(Logger):
-    def __init__(  # noqa: PLR0913
+    def __init__(
         self,
         run_name: Optional[str] = "dvclive_run",
         prefix="",
         log_model: Union[str, bool] = False,
         experiment=None,
-        dir: Optional[str] = None,  # noqa: A002
-        resume: bool = False,
-        report: Optional[str] = None,
-        save_dvc_exp: bool = False,
-        dvcyaml: bool = True,
-        cache_images: bool = False,
+        **kwargs,
     ):
         super().__init__()
         self._prefix = prefix
-        self._live_init: Dict[str, Any] = {
-            "resume": resume,
-            "report": report,
-            "save_dvc_exp": save_dvc_exp,
-            "dvcyaml": dvcyaml,
-            "cache_images": cache_images,
-        }
-        if dir is not None:
-            self._live_init["dir"] = dir
+        self._live_init: Dict[str, Any] = kwargs
         self._experiment = experiment
         self._version = run_name
-        if report == "notebook":
-            # Force Live instantiation
-            self.experiment  # noqa: B018
```

Comment:
This was added in https://github.com/iterative/dvclive/pull/432 but had to be updated after https://github.com/iterative/dvclive/issues/594. I don't think it's worthwhile since this still could cause issues in distributed compute. The only thing it does AFAICT is make sure the dvclive notebook report gets output at the very top of the cell output. 

@daavoo If you see this and have any more context, please chime in.

Example of output with this notebook-specific logic:

<img width="1116" alt="Screenshot 2023-10-19 at 3 07 31 PM" src="https://github.com/iterative/dvclive/assets/2308172/bc60b10c-0d70-4a36-9d18-79a894867f48">

Example without:

<img width="1121" alt="Screenshot 2023-10-19 at 3 11 02 PM" src="https://github.com/iterative/dvclive/assets/2308172/8d326c31-b842-4681-88ed-6cf04c23699b">

Reply:
Just to double check - will it break some existing Colabs / notebook examples?

Reply:
I tested it with the lightning colab example we have, which is what's shown above, so AFAIK it shouldn't break anything.